scribus/plugins/scriptplugin/cmdtext.cpp

```diff
@@ -1354,6 +1354,65 @@ PyObject *scribus_ispdfbookmark(PyObject* /* self */, PyObject* args)
 	return PyBool_FromLong(0);
 }
 
+PyObject *scribus_positiontopoint(PyObject* /* self */, PyObject* args)
+{
+	int pos;
+	char *Name = const_cast<char*>("");
+	if (!PyArg_ParseTuple(args, "i|es", &pos, "utf-8", &Name))
+		return nullptr;
+	if (!checkHaveDocument())
+		return nullptr;
+	PageItem *item = GetUniqueItem(QString::fromUtf8(Name));
+	if (item == nullptr)
+		return nullptr;
+	if (!(item->isTextFrame()) && !(item->isPathText()))
+	{
+		PyErr_SetString(WrongFrameTypeError, QObject::tr("Cannot insert text into non-text frame.","python error").toLocal8Bit().constData());
+		return nullptr;
+	}
+	if ((pos < 0) || (pos >= static_cast<int>(item->itemText.length())))
```

Comment:
from a quick check, itemText seems to be the story and `length()` the number of characters in the story, not in the frame. did you test the behavior when the position is in a linked frame on a different page or in the overflowing (hidden) part of the text?

Reply:
Changed to have this apply to just the text portion in the selected frame, and tested it with later linked frames.